src/cc1101.h

```diff
@@ -139,8 +139,8 @@ namespace cc1101 {
 	#endif
 #endif
 
-	#define wait_Miso()       while(isHigh(misoPin) ) { static uint8_t miso_count=255;delay(1); if(miso_count==0) return; miso_count--; }           // wait until SPI MISO line goes low
-	#define wait_Miso_rf()    while(isHigh(misoPin) ) { static uint8_t miso_count=255;delay(1); if(miso_count==0) return false; miso_count--; }     // wait until SPI MISO line goes low
+	#define wait_Miso()       do { uint8_t miso_count = 255; while(isHigh(misoPin)) { delay(1); if(miso_count == 0) return      ; miso_count--; } break; } while(1);    // wait until SPI MISO line goes low
+	#define wait_Miso_rf()    do { uint8_t miso_count = 255; while(isHigh(misoPin)) { delay(1); if(miso_count == 0) return false; miso_count--; } break; } while(1);    // wait until SPI MISO line goes low
```

Comment:
Was ist der Grund für die äußere Schleife?
Ich denke die läuft immer nur 1 mal bis zum break. Ein einfaches block statement sollte reichen.
Was denkst Du?

Reply:
```
{
    uint8_t miso_count = 255;
    while(isHigh(misoPin))
    {
        delay(1);
        if(miso_count == 0)
            return;
        miso_count--;
    }
}
```
Ja, sollte auch gehen. Eigentlich hatte ich ein do {} while(0) im Kopf, was das hier ja nicht ist - und es ist ja auch kein vollstaendiger Funktionsersatz. Ich koennte mit {} leben. Soll ich das aendern?

Reply:
Ja ich denke es führt so zu weniger Verwirrung was die Schleife soll.
Mit dieser Variante wird klarer, dass es nur darum geht die Variable in einen scope zu setzen.

Reply:
Mache ich - ich haue mir hier jetzt nochmal was Debugoutput rein um ein Gefuehl dafuer zu bekommen, wie schnell der Counter (mit static) auf 0 ist und daher nicht mehr funktioniert in meinem Fall. Und auch, wie sich der IO Pin Fix (mit den Highpullups) darauf auswirkt